share/dictionary.mikrotik

```diff
@@ -45,9 +42,9 @@ ATTRIBUTE	Mikrotik-Wireless-Comment		21	string
 ATTRIBUTE	Mikrotik-Delegated-IPv6-Pool		22	string
 ATTRIBUTE	Mikrotik-DHCP-Option-Set		23	string
 ATTRIBUTE	Mikrotik-DHCP-Option-Param-STR1		24	string
-ATTRIBUTE	Mikrotik-DHCP-Option-ParamSTR2		25	string
+ATTRIBUTE	Mikrotik-DHCP-Option-Param-STR2		25	string
```

Comment:
New attribute names can be added.  We shouldn't delete old names, as it will break existing configurations.